app/views/validations/_demo.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= form_with model: @user, class: "mt-4", data: { controller: "validations", action: "validations#submit" } do |f| %>
```

Comment:
As mentioned on Signal, I am intrigued by the functionality that LiveView devs get with their `phx-submit` attribute. I feel like between form/validation management and focus element stuffs, we could approach the idea of providing an optional JS include for projects that benefit from form management utilities. We can provide some opinionated structure without adding rigid pattern behaviors to the core library.

https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html#module-forms-and-input-handling